sites/game-client/map-renderer.mjs

```diff
@@ -66,19 +68,22 @@ class MapRenderer {
    * More precisely, the center of canvas will become (x, y).
    * If (x, y) is beyond the bound of the map, this function will clip the
    * coordinate to fit in the range of the map.
-   * @param {Number} x - The x coordinate (map coordinate).
-   * @param {Number} y - The y coordinate (map coordinate).
+   * @param {MapCoord} coord - The map coordinate.
+   * @param {Number} dx - Delta x to be added. Default is 0.
+   * @param {Number} dy - Delta y to be added. Default is 0.
    */
-  setViewerPosition(x, y) {
-    const mapSize = this.map.getMapSize();
+  setViewerPosition(coord, dx, dy) {
+    dx ??= 0;
```

Comment:
[Optional] I think some team members are still on older version of nodejs that doesn't support this. We should tell them to upgrade.

Reply:
I think Allen has used this syntax in previous commits. If everyone is okay with that, then there should be no concern here.